packages/core/src/agent/message-list/index.ts

```diff
@@ -2126,25 +2127,24 @@ export class MessageList {
         // Handle tool-invocation parts
         if (part.type === 'tool-invocation' && part.toolInvocation) {
           const inv = part.toolInvocation;
-          const v5State = inv.state === 'result' ? 'output-available' : 'input-available';
 
           if (inv.state === 'result' && inv.result) {
```

Comment:
@TylerBarnes If the tool intentionally returns null, could we end up setting the state to `input-available` accidentally?

Reply:
yeah we probably dont need the `inv.result` check at all